BandNameGenerator/ViewModel.cs

```diff
@@ -0,0 +1,54 @@
+﻿using BandNameGenerator.Genres;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BandNameGenerator
+{
+    public class ViewModel
+    {
+        public enum Genre
+        {
+            Metal,
+            Country,
+            Folk
+        }
+
+        public ViewModel()
+        {
+        }
+
+        private Genre selectedGenre = Genre.Metal;
+        public Genre SelectedGenre
+        {
+            get
+            {
+                return selectedGenre;
+            }
```

Comment:
I think this can be shortened to `get => selectedGenre`

Reply:
Indeed.